Tests/FrameworkTests/Satellite Tests/Text Fields Tests/SSNTextFieldTests.swift

```diff
@@ -81,13 +81,21 @@ class SSNTextFieldTests: VGSCollectBaseTestCase {
       ssnTextField.configuration = config
       
       let validSSN = [
-        "111111112", "222232222", "455555555",
-        "166666666", "899999999", "001010001",
-        "100123456", "143104563", "235231000",
-        "823423423", "665123455", "123456780",
-        "219099998", "078051125", "457555465",
-        /// mask should cut extra symbols
-        "234567890123456789"
```

Comment:
Why this test is removed?

Reply:
Hi @dmytrokhl, I'm using the same SSN info for testing that we are using in Android, but I think it make sense to add this value for testing.